apps/myaccount/src/components/consents/consent-edit.tsx

```diff
@@ -17,10 +17,17 @@
  */
 
 import { TestableComponentInterface } from "@wso2is/core/models";
+import { chain } from "lodash";
```

Comment:
Use default imports with lodash to reduce bundle size. Check other places as well.
```suggestion
import chain from "lodash/chain";
```

Reply:
I got this error while trying to apply this improvement:

```js
Uncaught TypeError: lodash_chain__WEBPACK_IMPORTED_MODULE_0___default(...)(...).map is not a function
    at isUpdatable (consent-edit.tsx:101)
    at AppConsentEdit (consent-edit.tsx:261)
    at renderWithHooks (react-dom.development.js:14797)
    at mountIndeterminateComponent (react-dom.development.js:17451)
    at beginWork (react-dom.development.js:18591)
    at HTMLUnknownElement.callCallback (react-dom.development.js:184)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:236)
    at invokeGuardedCallback (react-dom.development.js:287)
    at beginWork$1 (react-dom.development.js:23203)
    at performUnitOfWork (react-dom.development.js:22147)
```

For now I have decided to not to apply this improvement. But I will check on this and try to figure out why it didnt work.